codegen/projections/white_label/lib/white_label/params.rb

```diff
@@ -13,55 +13,55 @@ module Params
 
     class ClientError
       def self.build(params, context:)
-        Hearth::Validator.validate_types!(params, ::Hash, Types::ClientError, context: context)
+        Hearth::Validator.validate_types!(params, ::Hash, context: context)
```

Comment:
Why are we removing the ability to pass types?  I think thats a great feature, esp when using parts of an output.

Reply:
Because of input output shapes having different types, this does not work as expected. You still have to manipulate new input. I think hash only simplifies paths too.

Reply:
NVM. I think it does make sense for unions and sub structures..